apps/remote_control/lib/lexical/remote_control/search/store/backends/ets/schemas/v1.ex

```diff
@@ -25,6 +25,14 @@ defmodule Lexical.RemoteControl.Search.Store.Backends.Ets.Schemas.V1 do
     :path
   ]
 
+  defkey :by_prefix, [
+    :prefix,
+    :type,
+    :subtype,
+    :elixir_version,
+    :erlang_version
+  ]
```

Comment:
Are you referring to the "subject" in the key or the "subject" in the value? I think if we change the "subject" in the "by_subject" record key to a charlist instead of a binary, it would also achieve this goal. Currently, it is a binary.

https://github.com/lexical-lsp/lexical/blob/2b266ea7ff077cad578c73ee75d79c310a406864/apps/remote_control/lib/lexical/remote_control/search/store/backends/ets/schemas/v1.ex#L59

what we really need is something like this:

```elixir
 {:by_prefix, [70, 111, 111, 46, 66, 97, 114 | :_], :module, :definition,
 "1.15.7", "25.3.2.7"}
```

no matter is `by_prefix` or `by_subject`